src/components/organisms/ChatSidebar/components/PrivateChatPreview/PrivateChatPreview.module.scss

```diff
@@ -7,11 +7,11 @@
   display: flex;
   flex-direction: row;
   text-align: left;
+}
 
-  &__highlight {
-    .chatPreviewText {
-      font-weight: 600;
-    }
+.highlight {
+  .chatPreviewText {
```

Comment:
[NOTE] 
I think the modules are robust enough to not really need parent/child restrictions - we can just make all classes flat. The downside of this is, rare occasions like such would look like code smell and will be good to either be removed or commented with an explanation